frontend/src/pages/Assessments.js

```diff
@@ -1,8 +1,15 @@
 import React from "react";
 import Layout from "../components/Layout";
+import PromptSpeech from "../components/PromptSpeech";
 
 const Assessments = () => {
-  return <Layout>Assessments</Layout>;
+  return (
+    <Layout>
+      <h1>Assessments</h1>
```

Comment:
It doesn't matter for now since we need to change the text in the future too. But in the future, even the text doesn't have any styling, still make an attribute being like const Header = styled.h`` hehe. Same as the one below :0